README.md

```diff
@@ -7,101 +7,39 @@ has an identity layer (Azimuth), virtual machine (Vere), and operating system
 A running Urbit "ship" is designed to operate with other ships peer-to-peer.
 Urbit is a general-purpose, peer-to-peer computer and network.
 
-This repository contains:
+This repository contains the [Arvo Kernel][arvo]
 
-- The [Arvo OS][arvo]
-- [herb][herb], a tool for Unix control of an Urbit ship
-- Source code for [Landscape's web interface][land]
-- Source code for the [vere][vere] virtual machine.
-
-For more on the identity layer, see [Azimuth][azim]. To manage your Urbit
-identity, use [Bridge][brid].
-
-[arvo]: https://github.com/urbit/urbit/tree/master/pkg/arvo
-[azim]: https://github.com/urbit/azimuth
-[brid]: https://github.com/urbit/bridge
-[herb]: https://github.com/urbit/urbit/tree/master/pkg/herb
-[land]: https://github.com/urbit/urbit/tree/master/pkg/interface
-[vere]: https://github.com/urbit/urbit/tree/master/pkg/urbit
+For the Runtime, see [Vere][vere].
+For Landscape (the UI), see [Garden][gard].
```

Comment:
Wondering if we should specify with a bit more detail, like "the software management UI", or "the homepage UI". Although both of those are still too narrow, because it includes _backend_ software, for notifications, S3 settings, etc.

Also, are we _really_ doing "garden" as the repo name? Have we not suffered enough?? urbit/landscape is archived, we could obliterate it and replace it with ""garden"".

Reply:
Yeah, I can do that. We have the ok to move garden from the urbit org to tloncorp now, I can rename it to landscape as part of that.

Reply:
I'd remove this one entirely from the list, since it's not part of "Urbit" in the same way that these others are.